oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtAuthenticationConverter.java

```diff
@@ -65,4 +73,14 @@ public void setJwtGrantedAuthoritiesConverter(Converter<Jwt, Collection<GrantedA
 		Assert.notNull(jwtGrantedAuthoritiesConverter, "jwtGrantedAuthoritiesConverter cannot be null");
 		this.jwtGrantedAuthoritiesConverter = jwtGrantedAuthoritiesConverter;
 	}
+
+	/**
+	 * Sets the principal claim name.
+	 *
```

Comment:
Would you please indicate here what would be the default behavior if this method isn't called?

Reply:
Done.